webapp_v2/src/components/settings/Users/utils.ts

```diff
@@ -0,0 +1,8 @@
+import * as R from "ramda";
+
+export const RESERVED_USER_NAMES = ["admin"];
+export const RESERVED_GROUP_NAMES = ["admin"];
+
+export const sortByName = <T extends { name: string }>(list: T[]): T[] => {
```

Comment:
this is a nice generic utility that could be put in services/utils/index.ts

Reply:
Effectively

Reply:
Nah.. it's quite empty. Let put this in the main index.tsx

Reply:
Fixed